bravado/requests_client.py

```diff
@@ -185,18 +218,55 @@ def check_for_exceptions(self, response):
         except Exception as e:
             add_response_detail_to_errors(e)
 
+    def build_timeout(self, result_timeout):
+        """
+        Build the appropriate timeout object to pass to `session.send(...)`.
+
+        :param result_timeout: timeout that was passed into `result(..)`.
+        :return: timeout
+        :rtype: float or tuple(connect_timeout, timeout)
+        """
+        timeout = None
+        has_service_timeout = 'timeout' in self.misc_options
+        has_result_timeout = result_timeout is not None
```

Comment:
As result_timeout is being passed from [HttpFuture](https://github.com/analogue/bravado/blob/pass_timeouts/bravado/http_future.py#L32) `result_timeout` will always have some value (or a default value of 5.0), so `has_result_timeout` flag is always true.